projects/plugins/jetpack/modules/subscriptions/newsletter-widget/class-jetpack-newsletter-dashboard-widget.php

```diff
@@ -89,6 +89,13 @@ public static function wp_dashboard_setup() {
 			return;
 		}
 
+		// This covers both P2 and P2020 themes.
+		$site_id = \Jetpack_Options::get_option( 'id' );
+		$is_p2   = str_contains( get_stylesheet(), 'pub/p2' ) || function_exists( '\WPForTeams\is_wpforteams_site' ) && is_wpforteams_site( $site_id );
```

Comment:
Namespace is probably missing...

```suggestion
		$is_p2   = str_contains( get_stylesheet(), 'pub/p2' ) || function_exists( '\WPForTeams\is_wpforteams_site' ) && \WPForTeams\is_wpforteams_site( $site_id );
```